wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/MessageType.kt

```diff
@@ -246,19 +254,25 @@ data class MessageType(
       }
       val nestedTypes =
         messageElement.nestedTypes.map { get(packageName, protoType.nestedType(it.name), it, syntax) }
+
+      // namespace for nested extensions is fully-qualified name of enclosing message
```

Comment:
Please make sentences out of comments.

https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html